app/src/test/java/gov/va/vro/controller/MasControllerTest.java

```diff
@@ -87,27 +87,6 @@ void automatedClaim_validRequest() throws Exception {
     assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
   }
 
-  @Test
-  void automatedClaim_throwsException() throws Exception {
```

Comment:
This test no longer applies because the API returns immediatly

Reply:
Could you pls keep this test, we need throw an exception if the incoming parms are not in scope and don't have proper anchors in the claims